go/cmd/vtctldclient/command/vreplication/lookupvindex/lookupvindex.go

```diff
@@ -142,6 +151,18 @@ var (
 		RunE:                  commandCancel,
 	}
 
+	// complete makes a LookupVindexComplete call to a vtctld.
+	complete = &cobra.Command{
```

Comment:
I'm curious what the `complete` command gets us beyond what the `cancel` command does? Is it that `cancel` enforces that the vindex has NOT been externalized while `complete` enforces that it HAS been externalized? In either case, I would expect the command to delete the workflow if that check passes.

Reply:
Yes,  `complete` enforces that vindex has been externalized, but `cancel` simply deletes the workflow, regardless of whether vindex is externalized or internalized.